src/components/sidebar/Ship.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { ShipInterface } from '../../context/PlayerShipsContext';
```

Comment:
It appears that this is the only place where `ShipInterface` is used. Also, only `name` and `shipSections` are being used from `ShipInterface`. Could you define the interface in this file instead with the appropriate props?

Reply:
+ 1

Reply:
Ship interface is also used in the PlayerShipContext file, in the ShipsInterface. Should we move it to Ship.tsx and import to PlayerShipContext.tsx instead?

Reply:
It looks like `ShipInterface` when used in `PlayerShipsContext` behaves more like a type rather than an interface. I would suggest making `ShipInterface` a type, then creating a separate interface for the `Ship` component since it doesn't use all the fields.